### PR TITLE
fix: make sure that Zone does not show up in angular2.d.ts

### PR DESCRIPTION
**Pull request by mhevery**
- **Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- **What is the current behavior?** (You can also link to an open issue here)
- **What is the new behavior (if this is a feature change)?**
- **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
- **Other information**:

### CODE REVIEW

File: modules/angular2/src/core/zone/ng_zone_impl.ts [hunk 1]
```diff
@@ -45,52 +44,56 @@ export class NgZoneImpl implements ZoneSpec {
       if (trace) {
         this.inner = this.inner.fork(Zone['longStackTraceZoneSpec']);
       }
-      this.inner = this.inner.fork(this);
-    } else {
-      throw new Error('Angular2 needs to be run with Zone.js polyfill.');
-    }
-  }
+      this.inner = this.inner.fork({
```
Comment (alexeagle) [inline modules/angular2/src/core/zone/ng_zone_impl.ts]: nit: this gets pretty deeply nested, did you consider a private inner class that implements ZoneSpec?

File: modules/angular2/test/i18n/message_spec.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ import {
 import {Message, id} from 'angular2/src/i18n/message';
 
 export function main() {
-  ddescribe('Message', () => {
+  describe('Message', () => {
```
Comment (alexeagle) [inline modules/angular2/test/i18n/message_spec.ts]: does this have anything to do with us missing the bad release? or are we still missing test coverage?

File: gulpfile.js [hunk 3]
```diff
@@ -988,7 +988,7 @@ gulp.task('test.typings',
           ['!pre.test.typings.layoutNodeModule', '!pre.test.typings.copyTypingsSpec'], function() {
             var tsc = require('gulp-typescript');
 
-            return gulp.src([tmpdir + '/*.ts', 'node_modules/zone.js/dist/zone.js.d.ts'])
```
Comment (alexeagle) [inline gulpfile.js]: oh, this is why the test was passing.

**Comment by robwormald** [comment]
LGTM, tested build locally, fixes failure in .10

File: modules/angular2/test/i18n/message_spec.ts (hunk 2 above)
Comment (mhevery) [inline modules/angular2/test/i18n/message_spec.ts]: It is a separate SHA. We were only running single test on Travis.

File: modules/angular2/src/core/zone/ng_zone_impl.ts (hunk 1 above)
Comment (mhevery) [inline modules/angular2/src/core/zone/ng_zone_impl.ts]: Yes, I thought about it, but this is cheaper in terms of number of bytes. If you feel strongly I can make the change.

Comment (alexeagle) [inline modules/angular2/src/core/zone/ng_zone_impl.ts]: Nope, don't feel strongly.

**Pull request closed without merging**